### PR TITLE
{bio}[system/system] MaxQuant v2.4.2.0

### PR DESCRIPTION
**Pull request by SethosII**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/MaxQuant/MaxQuant-2.4.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+version = '2.4.2.0'
+
+homepage = 'https://maxquant.org/maxquant'
+description = """MaxQuant is a quantitative proteomics software package designed for analyzing large
+ mass-spectrometric data sets. It is specifically aimed at high-resolution MS data. Several labeling
+ techniques as well as label-free quantification are supported."""
+
+toolchain = SYSTEM
+
+# requires registration at https://www.maxquant.org/download_asset/maxquant/latest
```
Comment (smoors) [inline easybuild/easyconfigs/m/MaxQuant/MaxQuant-2.4.2.0.eb:13]: @SethosII please re-add parameter `download_instructions`, see MaxQuant-2.2.0.0-GCCcore-11.2.0.eb

**Comment by SethosII** [comment]
Test report by @SethosII
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
ribnode012 - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2640 v2 @ 2.00GHz, Python 3.8.10
See https://gist.github.com/SethosII/ab8ae9fa3377508b7cece4ba5da7049f for a full test report.

**Comment by SethosII** [comment]
Test report by @SethosII
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
ribnode012 - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2640 v2 @ 2.00GHz, Python 3.8.10
See https://gist.github.com/SethosII/dc9c7b43ed5d37b04af000fd9ec01215 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @SethosII!

**Pull request merged**